torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -12,14 +12,14 @@ namespace jit {
 namespace tensorexpr {
 
 // Returns true if the TE fuser supports this conv2d.
-bool conv2dIsSupportedJit(const Node* node);
```

Comment:
Should we keep jit::Node const?

Reply:
That is a good point. We should definitely keep all data structures not belonging to NNC const. Modified that.